packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -18,6 +18,7 @@ import 'snack_bar.dart';
 
 const double _kFloatingActionButtonMargin = 16.0; // TODO(hmuller): should be device dependent
 const Duration _kFloatingActionButtonSegue = const Duration(milliseconds: 200);
+const Object _kScaffoldStorageIdentifier = const Object();
```

Comment:
We shouldn't use a `const Object` because those are deduplicated across your app.  Instead, we need something that's unique to this scaffold, so we should use `static final _kxxx = new Object()`.